Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -102,6 +102,15 @@ extension WebPageWebView {
         get { scrollView.bouncesHorizontally }
         set { scrollView.bouncesHorizontally = newValue }
     }
+
+    @_spi(CrossImportOverlay)
+    public var allowsMagnification: Bool {
+        get { !self._disablesDoubleTapToZoom }
```

Comment:
Should we instead have an internal `_allowsMagnification` on iOS, and the logic in the setter kept in `WKWebView` itself?

Reply:
sure, will change